prestoadmin/server.py

```diff
@@ -82,11 +86,191 @@ def new_sysnode_processor(node_info_rows):
 CONNECTOR_INFO_SQL = 'select catalog_name from system.metadata.catalogs'
 _LOGGER = logging.getLogger(__name__)
 
+DOWNLOAD_DIRECTORY = main_dir
+DEFAULT_RPM_NAME = 'presto-server-rpm.rpm'
+
+
+def _find_or_download_most_recent_presto_rpm():
+    newest_rpm_release = 'https://repository.sonatype.org/service/local/artifact/maven' \
+                         '/content?r=central-proxy&g=com.facebook.presto' \
+                         '&a=presto-server-rpm&e=rpm&v=RELEASE'
+    return _find_or_download_rpm(newest_rpm_release)
+
+
+def _check_if_valid_version(rpm_version):
```

Comment:
Document what this should and shouldn't match. Additionally: add unit tests if you haven't already.